### DISCUSSION TITLE
How can i load LightningLite model?

### DISCUSSION THREAD

**Discussion by morestart** [Lightning-AI/pytorch-lightning#14935] [Lightning Trainer API: Trainer, LightningModule, LightningDataModule]
In pytorchLightning, i can use model.load_fromcheckpoint to load my model.

but i can't find any ways to load lightningLite model

i save model use ` self.save(model.state_dict(), os.path.join(model_save, f'{best_loss:.3f}-{epoch}.pt'))`

**Comment by morestart** [comment]
okay, i can't find any docs about load lightningLite model, i will back to the original ways to train my model.

**Comment by akihironitta** [comment]
Hi @morestart, it is documented at https://pytorch-lightning.readthedocs.io/en/latest/model/build_model_expert.html#load:

```python
# Instead of `torch.save(...)`, call:
self.save(model.state_dict(), "path/to/checkpoint.ckpt")

# Instead of `torch.load(...)`, call:
self.load("path/to/checkpoint.ckpt")
```

**Reply by morestart** [reply]
i think this is not a good idea to load my model, you mean i need to create a inheritance class to load my model?
